src/prefect/schedules/clocks.py

```diff
@@ -65,6 +67,8 @@ class Clock:
             Flow Runs which are run on this clock's events
         - labels (List[str], optional): a list of labels to apply to all flow runs generated
             from this Clock
+        - flow_run_name_template (str, optional): a templated string referencing any parameterized
+            values which the scheduler can use to create scheduled flow runs
```

Comment:
Is it worth discussing the available values a bit more and what happens when this is `None`?

Reply:
If it's `None`, the current logic will kick in which sets a random adj-animal run name.  

In terms of available values, it's a good question - I don't think I see a use case for templating the name based on properties that are also known at Clock creation time.  It seems hard-coded names here would allow for the same amount of functionality?

Reply:
> If it's `None`, the current logic will kick in which sets a random adj-animal run name.

I mean is that worth explaining in the docstring :)

Reply:
@cicdw your proposal is that we remove templating in favor of a static string? I think we should offer some dynamism to this even if it's not parameters (though I think there's a value in only writing your parameter value once instead of multiple times when it changes).

Reply:
Yea, and in fact I actually think we should resolve https://github.com/PrefectHQ/server/issues/125 first; that way we can template via scheduled start time, parameters for the run (which might include flow group defaults once that issue is closed), and also labels for the run (which could come from flow group defaults as well)